.eslintrc.js

```diff
@@ -2,5 +2,11 @@ module.exports = {
   root: true,
   parser: "@typescript-eslint/parser",
   plugins: ["@typescript-eslint"],
+  parserOptions: {
+    project: "./tsconfig.json",
+  },
+  rules: {
+    "@typescript-eslint/no-misused-promises": "error",
```

Comment:
The promise inside `onProgramAccounts` wasn't handled, and this rules would've caught